easybuild/easyconfigs/g/GATE/GATE-8.2_fix_define_min_max.patch

```diff
@@ -0,0 +1,14 @@
+diff -Nru Gate-8.2.orig/source/general/include/GateIAEAUtilities.h Gate-8.2/source/general/include/GateIAEAUtilities.h
```

Comment:
Please add a description and author information to the patch

Reply:
done